test-utils/state-viewer/src/cli.rs

```diff
@@ -109,7 +109,13 @@ pub enum StateViewerSubCommand {
 
 impl StateViewerSubCommand {
     pub fn run(self, home_dir: &Path, genesis_validation: GenesisValidationMode) {
-        let near_config = load_config(home_dir, genesis_validation);
+        let near_config = match load_config(home_dir, genesis_validation) {
+            Ok(c) => c,
+            Err(e) => {
+                tracing::error!("Error loading config: {:#}", e);
+                return;
+            }
+        };
```

Comment:
I think it might be better to `.unwrap()` rather than `return` here and elsewhere, such that the exit code is non-zero.

Reply:
done. @matklad what do you think about returning a result from main(), maybe in another PR. just so we can exit normally here and not have to panic to get a nonzero exit code

Reply:
Yeah, we should have 

```
fn main() -> anyhow::Result<()>
```

Reply:
> ```
> fn main() -> anyhow::Result<()>
> ```

Or even better:
```rust
fn main() -> Result<(), anyhow::Error>
```

Reply:
Hm, I'd say that `anyhow::Result<()>` would be better. That is

* both `Result<(), anyhow::Error>` and `anyhow::Result<()>` mean exactly the same, they are different spellings of the same type.
* but `anyhow::Result<()>` is a preferred spelling:
  * it's a bit more idiomatic (compare, ig, `io::Result` or `fmt::Result`)
  * it's just reads better from "qualified names" point of view -- `anyhow` is the prefix, rather than the middle. 

No strong feelings here though!

Reply:
To expand on this: 

* yes, `Result<(), anyhow::Error>` is a bit easier to understand, as it is more direct, you don't have to know about `foo::Result<T>` idiom
* but this is outweighed by qualified result form being idiomatic -- code generally should be written in idiomatic form for consistency, and because idioms, while taking some time to learn, do generally make the code better. 

For qualified form being idiomatic:

* example from stdlib https://doc.rust-lang.org/stable/std/io/index.html (io::Result)
* documentation for the idiom https://blog.burntsushi.net/rust-error-handling/#the-result-type-alias-idiom
* the alias that exists in anyhow

Reply:
@matklad I see, personally I think that `Result<(), anyhow::Error>` is better.

However, on the other hand, Rust std already uses `io::Result`, `fmt::Result`. I agree that it's better to be consistent. Most often consistent approach is preferred, even though it may not be an optimal approach.
In that case, let's use `anyhow::Result<()> `